### PR TITLE
Add io.github.pragha_music_player

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/pragha-music-player/pragha/issues/117
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://github.com/pragha-music-player/pragha/pull/173

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build io.github.pragha_music_player

File: io.github.pragha_music_player.yml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+rename-icon: pragha
+rename-desktop-file: pragha.desktop
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=home
```
Comment (hfiguiere) [inline io.github.pragha_music_player.yml:15]: why not `xdg-music`

File: io.github.pragha_music_player.yml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    sources:
+      - type: archive
+        url: https://github.com/taglib/taglib/releases/download/v1.12/taglib-1.12.tar.gz
+        sha256: 7fccd07669a523b07a15bd24c8da1bbb92206cb19e9366c3692af3d79253b703
+
+  - name: libcdio
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://ftp.gnu.org/gnu/libcdio/libcdio-2.1.0.tar.bz2
```
Comment (hfiguiere) [inline io.github.pragha_music_player.yml:110]: https maybe?

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: io.github.pragha_music_player.yml [hunk 3]
```diff
@@ -0,0 +1,144 @@
+app-id: io.github.pragha_music_player
```
[the remark this answers is not in the document]
Comment (matiasdelellis) [reply io.github.pragha_music_player.yml:1]: ..the id 'io.github.pragha-music-player', is the reverse-dns of the http://pragha-music-player.github.io/ domain that works..

But...
>    https://dbus.freedesktop.org/doc/dbus-specification.html#message-protocol-names
>    If the author's DNS domain name contains hyphen/minus characters ('-'), which are not allowed in D-Bus interface names, they should be replaced by underscores.

File: io.github.pragha_music_player.yml (hunk 1 above)
Comment (Mailaender) [reply io.github.pragha_music_player.yml:15]: It let's you select folders. Might be weird if it does not work in `$HOME`?

**Comment by Mailaender** [comment]
bot, build io.github.pragha_music_player

File: io.github.pragha_music_player.pragha.yml [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+                x-checker-data:
+                  type: anitya
+                  project-id: 15888
+                  stable-only: true
+                  url-template: https://archive.xfce.org/src/xfce/xfce4-dev-tools/$major.$minor/xfce4-dev-tools-$version.tar.bz2
+        cleanup:
+          - /bin
+          - /share/bash-completion
+        config-opts:
+          - --enable-maintainer-mode
```
Comment (hfiguiere) [inline io.github.pragha_music_player.pragha.yml:66]: you sure this is needed?

File: io.github.pragha_music_player.pragha.yml [hunk 5]
```diff
@@ -0,0 +1,144 @@
+app-id: io.github.pragha_music_player.pragha
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: pragha
+rename-icon: pragha
+rename-desktop-file: pragha.desktop
```
[the remark this answers is not in the document]
Comment (Mailaender) [reply io.github.pragha_music_player.pragha.yml:7]: Last time you made a huge fuss about me not patching upstream.

File: io.github.pragha_music_player.pragha.yml (hunk 4 above)
Comment (Mailaender) [reply io.github.pragha_music_player.pragha.yml:66]: I copied this from mousepad.

Comment (hfiguiere) [reply io.github.pragha_music_player.pragha.yml:66]: and?

Comment (Mailaender) [reply io.github.pragha_music_player.pragha.yml]: Let's get rid of it.

**Comment by Mailaender** [comment]
bot, build io.github.pragha_music_player.pragha

**Comment by Mailaender** [comment]
bot, build io.github.pragha_music_player.pragha

**Comment by Mailaender** [comment]
bot, build io.github.pragha_music_player

**Comment by Mailaender** [comment]
bot, build io.github.pragha_music_player.pragha

**Comment by Mailaender** [comment]
bot, build io.github.pragha_music_player.pragha

**Pull request closed without merging**